### ISSUE TITLE
Upgrade to Logstash 2

### ISSUE THREAD

**Issue by sherzberg** [elastic/logstash-test-helpers#1]
This project looks great! Any plans for logstash 2? Our logstash config is becoming unwieldy and we want to start using this.

**Comment by purbon** [comment]
Yeah! I should update it to be 2.0, for now this is a work in progress project. Feel free to provide a PR to do the update, should be as easy as changing the .gemspec :+1:
